h2o-algos/src/main/java/water/rapids/prims/AstPredictedVsActualByVar.java

```diff
@@ -0,0 +1,120 @@
+package water.rapids.prims;
+
+import hex.Model;
+import water.MRTask;
+import water.fvec.*;
+import water.rapids.Env;
+import water.rapids.ast.AstPrimitive;
+import water.rapids.ast.AstRoot;
+import water.rapids.vals.ValFrame;
+import water.util.ArrayUtils;
+
+import java.util.Arrays;
+
+public class AstPredictedVsActualByVar extends AstPrimitive<AstPredictedVsActualByVar> {
+
+    @Override
+    public String[] args() {
+        return new String[]{"model"};
+    }
+
+    @Override
+    public int nargs() {
+        return 1 + 4;
+    } // (predicted.vs.actual.by.var model frame variable predicted)
+
+    @Override
+    public String str() {
+        return "predicted.vs.actual.by.var";
+    }
+
+    @Override
+    public ValFrame apply(Env env, Env.StackHelp stk, AstRoot[] asts) {
+        Model<?, ?, ?> model = (Model<?, ?, ?>) stk.track(asts[1].exec(env)).getModel();
+        if (!model.isSupervised()) {
+            throw new IllegalArgumentException("Only supervised models are supported for calculating predicted v actual");
+        }
+        if (model._output.isMultinomialClassifier()) {
```

Comment:
what about binomial?  Shouldn't it be isMultinomialClassifier && is BinomialClassifier?